### PR TITLE
Update speech usage examples.

### PR DESCRIPTION
**Pull request by daspecster**
Towards: #3172

Test script: https://gist.github.com/daspecster/718d01c7896ee576743292549cfb6927

### CODE REVIEW

File: docs/speech-usage.rst [hunk 1]
```diff
@@ -89,15 +89,12 @@ Great Britian.
     >>> sample = client.sample(source_uri='gs://my-bucket/recording.flac',
     ...                        encoding=speech.Encoding.FLAC,
     ...                        sample_rate=44100)
-    >>> results = sample.sync_recognize(
-    ...     speech.Encoding.FLAC, 16000,
-    ...     source_uri='gs://my-bucket/recording.flac', language_code='en-GB',
-    ...     max_alternatives=2)
+    >>> results = sample.sync_recognize(language_code='en-GB', max_alternatives=2)
```
Comment (dhermes) [inline docs/speech-usage.rst]: Seems too long?

**Comment by daspecster** [comment]
@dhermes yeah I think it would be. @lukesneeringer sounded like I should just run through these manually for now but I'm game either way.

@lukesneeringer WDYT?

**Comment by dhermes** [comment]
These "hand-crafted" code snippets will always be vulernable to bit-rot if we don't have a way to test them.

**Comment by lukesneeringer** [comment]
Speech is not a GA candidate, so if you are done checking off the GA requirements, feel free to take a more involved pass and make doctests.

**Comment by lukesneeringer** [comment]
To be clear, though, please make sure everything is pristine on the GA libraries first.

**Comment by daspecster** [comment]
@lukesneeringer @dhermes moved back into the original issue.
https://github.com/GoogleCloudPlatform/google-cloud-python/issues/3172#issuecomment-289141318

**Comment by lukesneeringer** [comment]
Yes.

**Comment by daspecster** [comment]
@lukesneeringer is this good to merge?

**Pull request merged**